config/environment.js

```diff
@@ -113,11 +114,11 @@ module.exports = function(environment) {
   };
 
   if (environment === "development") {
-    ENV.APP.API_HOST_URL = "http://localhost:3000";
+    ENV.APP.API_HOST_URL = "http://localhost:4000";
```

Comment:
remove this

Reply:
@abulsayyad123, added this purposefully so that we dont have to change it everytime.